webapp/src/test/java/com/box/l10n/mojito/service/assetintegritychecker/integritychecker/PrintLikeVariableTypeIntegrityCheckerTest.java

```diff
@@ -0,0 +1,103 @@
+package com.box.l10n.mojito.service.assetintegritychecker.integritychecker;
+
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class PrintLikeVariableTypeIntegrityCheckerTest {
+
+    @Test
+    public void testCheckPassesIfTypeVariablesMatch() throws PrintfLikeIntegrityCheckerException {
+        PrintfLikeVariableTypeIntegrityChecker checker = new PrintfLikeVariableTypeIntegrityChecker();
+        String source = "%(count)s view";
+        String target = "%(count)s view";
```

Comment:
btw, do you know what format that is? is it standard?

Reply:
@aurambaj That style is used in python: https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting